asset/errors.go

```diff
@@ -11,9 +12,41 @@ var (
 )
 
 type NotFoundError struct {
-	AssetID string
+	AssetID   string
+	AssetType string
```

Comment:
CMIIW If we don't need `AssetRepository.GetIDByURN` anymore, then we do not need `AssetType` and `AssetURN` here as well.